Worktrolly/worktrolly
# feat(Patch1): updates counters and fixes sprint documents' structure

### Functionality:
A patch for updating counters 

### Solution:
Created patch1 component which updates the counters based on all tasks. Gave a similar structure to backlog and deleted sprints to match the other sprints in db.

### Risk level:

### How to test:
1. firebase emulators:start
2. ng serve
2. Set up organization and tasks
3. Go to patch1 component through link
4. Enter OrgDomain, OrgId and TeamId
5. Notice the update in counters
